automation-roles/10-validation/lint-config/scripts/pre-execution-processor.py

```diff
@@ -28,8 +32,10 @@
 # attributes_updated: <dict>
 # errors: []
 
-
-result = preprocessor(attributes=generatorAttributes, fullConfig=generatorFullConfig)
+if args.platform == "aws":
```

Comment:
The extra parameters should not be dependent on the cloud platform. If required update all `preprocessor.py`  scripts to accept the additional parameter, even if not used.

Reply:
@fketelaars , Thanks. I will take a look at the issue you found, and update all `preprocessor.py` scripts. `preprocessor.py` script will only apply for new validation way for aws.

Reply:
Please make sure it applies to all validations, so update all preprocessor.py scripts and correct the parameters.

Reply:
@fketelaars, now only aws platform will apply for new validations. For other platform, it still apply for previous validations even if I updated all preprocessor.py scripts to accept one more parameter.   If aws new validations are expected, I will update code for other platforms.  Thanks.

Reply:
Looks good now. Thanks.